commands/access/index.js

```diff
@@ -7,12 +7,15 @@ let Utils     = require('../../lib/utils');
 let co        = require('co');
 let orgFlags;
 
+function orgHasGranularPermissions(orgFlags) {
+  return ((orgFlags.indexOf('org-access-controls') !== -1) || (orgFlags.indexOf('static-permissions') !== -1));
```

Comment:
We're using lodash in CLI, right? Might be cleaner to use the "includes" method here. https://lodash.com/docs#includes